src/lib/style/base/_core.scss

```diff
@@ -143,6 +143,12 @@ svg.SRC-hoverBox {
       }
     }
   }
+  .SRC-primary-background-color {
+    svg,
+    path {
+      fill: #fff;
+    }
+  }
```

Comment:
I don't think this is used in very many places, so it should be fine. It might be worth explicitly handling this behavior in IconSvg so we can properly style fill/color depending on what's appropriate.